### DISCUSSION TITLE
Can i test my code on Visual Studio Code?

### DISCUSSION THREAD

**Discussion by omr-dev** [codewars/codewars.com#2582] [Q&A] [answered]
Hi,
I want to write and test my code on my Computer, than i am going to use copy+paste for codewars. Is it possible?

**Comment by hobovsky** [comment, marked as answer]
Unfortunately, currently there's no way to fetch the code from Codewars to your local IDE. You need to copy your solution and sample tests into your project, but then you can work on it locally.
The exact setup depends on language, it looks differently for Python, JS, or C#. But you still need to copy the code from kata trainer to your IDE, and then copy your solution form your IDE to the trainer and submit it.
